src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -249,7 +249,7 @@ FormPage.propTypes = {
   route: PropTypes.shape({
     pageConfig: PropTypes.shape({
       arrayPath: PropTypes.string,
-      CustomPage: PropTypes.element,
+      CustomPage: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
```

Comment:
@jamigibbs I just added this PropTypes update for `CustomPage`. Without it, we see a type mismatch in the console for `CustomPage` functions